packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  withStyles,
+  Theme,
+  WithStyles,
+  Grid,
+  createStyles,
+} from '@material-ui/core';
+import { StyleRules } from '@material-ui/core/styles';
+import Image from '../images/Jasmin4 _DSC7054.jpg';
+import DatagatewayLogoWithText from '../images/datagateway-logo-white.svg';
+import ExploreImage from '../images/explore.jpg';
+import DiscoverImage from '../images/discover.jpg';
+import DownloadImage from '../images/download.jpg';
+// import { UKRITheme } from './theming';
+import homePageContents from './contents';
+
+const styles = (theme: Theme): StyleRules =>
+  createStyles({
+    bigImage: {
+      backgroundImage: `url("${Image}")`,
+      height: 250,
+      width: '100%',
+      '& img': {
+        paddingLeft: 90,
+        paddingTop: 80,
+        height: 150,
+        float: 'left',
+      },
+    },
+    howItWorks: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      paddingLeft: '10vw',
+      paddingRight: '10vw',
+      paddingTop: 15,
+      backgroundColor: theme.palette.background.default,
+    },
+    howItWorksTitle: {
+      fontWeight: 'bold',
+      color: theme.palette.text.primary,
+      paddingBottom: 20,
+    },
+    howItWorksGridItem: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    howItWorksGridItemTitle: {
+      // color: (theme as UKRITheme).ukri.bright.orange,    // TODO add this back. Cannot find 'bright' of undefined
```

Comment:
Is `UKRITheme` something that needs to be exported from dg-common in `index.tsx` to get this to work? FWIW this code works fine in SciGateway's homepage

Reply:
Leaving the theming file out as it wasn't worth importing the whole thing if only one line was taken from it